.claude/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "permissions": {
+    "allow": [],
+    "deny": [
+      "mcp__go-language-server__edit_file"
```

Comment:
Is there a particular reason to deny this one? Seems like it could be useful for larger refactors.